test/e2e/tests/test_external_oidc.py

```
@@ -112,3 +147,69 @@ def test_minted_api_key_can_list_models_and_infer(self, maas_api_base_url: str):
+
+        response_json = models_response.json()
+        assert response_json.get("object") == "list", f"Expected object=list, got: {response_json}"
+
+        items = response_json.get("data", [])
+        # Should return empty list (not null)
+        assert isinstance(items, list), f"Expected data to be a list, got {type(items)}"
+        assert len(items) == 0, (
+            f"Expected empty list for user without group access, got {len(items)} model(s)"
+        )
+
+        print(f"[oidc] User without group access correctly received empty list (200 OK)")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)